Source/DOM classes/Unported or Partial DOM/SVGTextElement.m

```diff
@@ -144,22 +173,192 @@ - (CALayer *) newLayer
         label.anchorPoint = CGPointZero; // WARNING: SVG applies transforms around the top-left as origin, whereas Apple defaults to center as origin, so we tell Apple to work "like SVG" here.
     
 	label.affineTransform = textTransformAbsoluteWithLocalPositionOffset;
-	label.fontSize = effectiveFontSize;
-    label.string = effectiveText;
+    label.string = [attributedString copy];
```

Comment:
`CATextLayer` is just like a `UITextView`. No detailed cusotmization. Using `NSAttributedString` is much better than String because it supports stroke via `NSStrokeWidthAttributeName` && `NSStrokeColorAttributeName`

For now, this stroke can only supports color, but not gradient layer mask. I think maybe someone can rewrite this to using Core Text API to draw each glyph character on the screen and mix gradient. Which need far more code.